README.rst

```diff
@@ -9,6 +9,16 @@ Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/about>`_.
 Currently, Windows PowerShell, Command Prompt, and Git Shell for
 Windows are supported with limitations (See `Windows Issues`_).
 
+This product is supported by the Cybersecurity Development team at the 
+University of Illinois, on a best-effort basis. As of the last update to 
+this README, the expected End-of-Life and End-of-Support dates of this 
+product are 05 October 2025.
+
+The End-of-Life was decided upon based on these dependencies::
```

Comment:
Do we need to give a reason? If we do then I think we can shorten it, just say in the paragraph above:

the expected End-of-Life and End-of-Support dates of this product is October 5, 2025, the same as its primary dependences the AWSCLI and Python V3.9.
 
An inline link to the the EOL for each would be nice as well.

Reply:
I like the separation of the dates as individual items. It draws attention to the details. 

I am not sure that we need to provide a link for other EOL information - providing these dates at all is a courtesy for the User.

Reply:
+ I did go ahead and make a pass at tightening up the language, and I'm satisfied with how it turned out. I think we'll keep the more detailed version for our knowledge base article as a reminder to take these on a case-by-case basis.

+ I have added the link, as it is pretty hard to find. (As we chatted about Googling any product name and "end of life" leads to a lot of un-related information.

+ This tool was incredibly helpful. Great find!
https://endoflife.date/

+ I have removed the 5th from the dates, as the official Python page only talks about October of 2025.